plane/source/docs/tuning-cruise.rst

```diff
@@ -10,32 +10,54 @@ Overview of Constant Altitude, Level Flight Operation
+During :ref:`common-accelerometer-calibration`, a "level" attitude position is set with the wings and fuselage perfectly level. A plane can fly holding a constant altitude at range of speeds, depending on throttle level and Angle of Attack (AOA).
 
-- **(Preferred)** Add a few degrees nose up to :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>`. This will also add an offset to the artificial horizon on a GCS or an OSD.
-- Position vehicle with a few degrees nose up during the first, Level step of accelerometer calibration.
-- Position vehicle with a few degrees nose up and use the  ``Calibrate Level`` button on the Mission Planner page to adjust the AHRS_TRIM parameters. AHRS_TRIM parameters can only change the difference between the autopilot's plane and "level" by 10 degrees maximum. If more is needed, (e.g. the autopilot is mounted slightly downward), then you can use :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>` to alter the AOA manually.
-- Increase :ref:`TRIM_ARSPD_CM<TRIM_ARSPD_CM>`. (Increase :ref:`TRIM_THROTTLE<TRIM_THROTTLE>` if an airspeed sensor or synthetic airspeed is not used)
+.. image:: ../../../images/AOA.jpg
 
-While the autopilot will operate correctly with minor offsets to the calibrated "level" attitude, optimum performance will occur if this is adjusted to match the pitch attitude that the autopilot tends to use in automatic throttle modes. This can be done by examining the average ``ATT.pitch`` value during level altitude flight from dataflash logs or in-flight on an artificial horizon (e.g. GCS or OSD HUD). Make slight adjustments to the :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>` parameter value based on the value seen on the artificial horizon. Making these adjustments also will allow STABILIZE and FBWA modes to hold altitude at the same throttle level as used in the other modes.
+The AOA is usually several degrees but varies depending on cruise speed/throttle. This is the "trim level" pitch. This is explained in the diagrams above and is known as the "trim level" condition. While many planes will have some Angle of Incidence (i.e. the cord of the wing is at a positive angle to the fuselage cord) built-in, some do not, and some need a slightly higher AOA to fly at desired cruise speeds.
 
-.. image:: ../../../images/AOA.jpg
+If the level step of calibration is done with the plane's fuselage line level, initial flights will be safe, but the aircraft may not hold altitude at the desired flying speed in non-altitude controlled modes (i.e. fly too fast and/or require too much throttle to hold altitude) and/or the cruising speed in non airspeed sensor equipped planes may be faster or slower than desired.
```

Comment:
Move the closing parenthesis to the end.

Reply:
yep...what I meant

Reply:
reworded